examples/simple/simple_push_pull.go

```diff
@@ -50,7 +50,11 @@ func main() {
 	memoryStore := content.NewMemory()
 	desc, err := memoryStore.Add(fileName, customMediaType, fileContent)
 	check(err)
-	_, err = memoryStore.GenerateManifest(ref, nil, desc)
+
+	config, configDesc, manifest, manifestDesc, err := content.GenerateManifestAndConfig(nil, nil, desc)
```

Comment:
This is what is causing the tests to fail:
```suggestion
	manifest, manifestDesc, config, configDesc, err := content.GenerateManifestAndConfig(nil, nil, desc)
```

Reply:
This is what I meant with the order of the returned vars for `GenerateManifestAndConfig` in the simple example being wrong.

Reply:
Nice catch. And I explicitly put the named return vars in the signature to make it easier to avoid that error.